### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/content_conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#221]
```
I have Added in Italian translation
conversationlist_vilegard_erttu
conversationlist_vilegard_villagers
conversationlist_kaori
conversationlist_umar
conversationlist_fallhaven_warden
```

Original issue reported on code.google.com by `k6b...@gmail.com` on 26 Jul 2011 at 8:47

Attachments:
- [content_conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-221/comment-0/content_conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, committed in r143.
```

Original comment by `oskar.wi...@gmail.com` on 26 Jul 2011 at 3:20
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.10**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
